docs/examples/general/data_loading/dataloading_webdataset.ipynb

```diff
@@ -0,0 +1,200 @@
+{
```

Comment:
<blockquote>Processing images involves adding artificial noise and <code>HWC</code> -&gt; <code>CHW</code> conversion process</blockquote><p> </p><p><em>This makes it sound like a general rule, not like a description of this particular pipeline. How about:</em></p><p> </p><p><strong>In this example, we augment the data by applying noise and blurring.</strong></p><p> </p><p><em>By the way, this pipeline doesn't model a real-life image acquisition - also, without a resize or crop to a common shape, it's not ready for ingestion by most NNs</em></p><p> </p><p><em>Also, where does the conversion to CHW happen? I don't see it here...</em></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA/DALI/pull/3372/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>